internal/service/evidently/wait.go

```diff
@@ -0,0 +1,59 @@
+package evidently
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/service/cloudwatchevidently"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func waitProjectCreated(conn *cloudwatchevidently.CloudWatchEvidently, nameOrARN string, timeout time.Duration) (*cloudwatchevidently.GetProjectOutput, error) {
+	stateConf := &resource.StateChangeConf{
+		Pending: []string{},
+		Target:  []string{cloudwatchevidently.ProjectStatusAvailable},
+		Refresh: statusProject(conn, nameOrARN),
+		Timeout: timeout,
+	}
+
+	outputRaw, err := stateConf.WaitForState()
+
+	if output, ok := outputRaw.(*cloudwatchevidently.GetProjectOutput); ok {
```

Comment:
We should assert this output to `*cloudwatchevidently.Project` to match the return type of the finder.

```suggestion
	if output, ok := outputRaw.(*cloudwatchevidently.Project); ok {
```